tests/console/suseconnect_scc.pm

```diff
@@ -34,7 +34,9 @@ sub run {
 
     select_console 'root-console';
     assert_script_run "SUSEConnect --url $scc_url -r $reg_code";
-    assert_script_run 'SUSEConnect --list-extensions';
+    my $expected_suseconnect_output = 'SUSEConnect -p';
+    $expected_suseconnect_output = 'transactional-update register -p' if (get_var('SYSTEM_ROLE') =~ /microos|serverro/);
```

Comment:
hm, how about a simple conditional assignment in the declaration line?:

```perl
my $transactional = get_var('SYSTEM_ROLE') =~ /microos|serverro/
my $expected_suseconnect_output = $transactional ? 'transactional-update register -p' : 'SUSEConnect -p';
```

Reply:
Actually, with your suggestion I just realize that I still need here a function "is_transactional_system".